lib/ash_json_api/api/router.ex

```diff
@@ -54,6 +54,12 @@ defmodule AshJsonApi.Api.Router do
           to: AshJsonApi.Controllers.Schema,
           init_opts: [api: api]
         )
+
+        match("/schema.json",
+          via: :get,
+          to: AshJsonApi.Controllers.Schema,
+          init_opts: [api: api]
+        )
```

Comment:
I'm not confident this is required, remove if you think it's unnecessary.

Reply:
I don't see a reason not to support it also.